www/js/app.js

```diff
@@ -949,18 +952,25 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                 });
             });
         }
+        
+        function loadNoScriptTags() {
+            // For each noscript tag, we replace it with its content, so that the browser interprets it
+            $('#articleContent').contents().find('noscript').replaceWith(function () {
+                return this.textContent;
```

Comment:
Is there a reason to use `textContent` rather than `innerHTML`? Do we want to interpret the HTML or filter it out in some way? It's a genuine question, because I don't know what normally goes inside noscript tags, or quite what the difference is between `textContent`, `innerText`, jQuery's `text`, etc.